apps/masterbots.ai/components/routes/chat/chat-chatbot-details.tsx

```
@@ -1,107 +1,166 @@
 
-  //* Fetches the name of the active category
   const getCategoryName = async () => {
-    const category = await getCategory({ categoryId: activeCategory as number }) //* Retrieves category details
-    setCategoryName(category.name) //* Updates category name state
+    const category = await getCategory({ categoryId: activeCategory as number })
+    setCategoryName(category.name)
   }
 
-  // biome-ignore lint/correctness/useExhaustiveDependencies: <explanation>
+  // eslint-disable-next-line react-hooks/rules-of-hooks
   useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: getCategoryName

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)